Sources/Views/InputBarItem.swift

```diff
@@ -112,11 +127,12 @@ open class InputBarButtonItem: UIButton {
         }
     }
     
-    // MARK: - Hooks
+    // MARK: - Reactive Hooks
     
     private var onTouchUpInsideAction: InputBarButtonItemAction?
     private var onKeyboardEditingBeginsAction: InputBarButtonItemAction?
     private var onKeyboardEditingEndsAction: InputBarButtonItemAction?
+    private var onKeyboardSwipeGestureAction: ((InputBarButtonItem, UISwipeGestureRecognizer) -> Void)?
```

Comment:
Let's just make sure this is all good. If I remember correctly it was causing some weird behavior a few versions back. If we keep it, add it to the changelog as an `Added` entry 😃

Reply:
Oh no that shouldn't be there. was waiting till everything else is done to add swipe gestures again